examples/README.md

```diff
@@ -1,31 +1,31 @@
 # Live Examples to exercise this suite of TerraForm Modules
 
 ## Usage
-- Select the correct AWS credentials to use with sufficient priviledges to spin up the infrastructure, e.g. `export AWS_PROFILE=admin`
-- Create a backend to store terraform state if requires. There is an example provided under `examples/backend_int` that creates s3 bucket to store terraform state and dynamodb table to store terraform state lock. 
-- To create a full Open Data Cube (ODC) eks infrastructure on AWS platform, execute each modules defined under `examples/stage/<module>`. Please note that number in front, e.g. `01_`, represents it's order of execution to manage dependencies.
-- You require to adjust some of the configuration params such as `owner`, `namespace`, `environment`, `region` and `terraform backend` - unique to your organisation.
+- Select the correct AWS credentials to use with sufficient privileges to spin up the infrastructure, e.g. `export AWS_PROFILE=admin`
+- Create a backend to store terraform state if requires. There is an example provided under `examples/backend_int` that creates the s3 bucket to store terraform state and the dynamodb table to store terraform state lock. 
+- To create a full Open Data Cube (ODC) eks infrastructure on AWS platform, execute each module defined under `examples/stage/<module>`. Please note that the number in front, e.g. `01_`, represents the correct order of execution to manage dependencies.
+- You need to adjust some of the configuration params such as `owner`, `namespace`, `environment`, `region` and `terraform backend` - so they are unique to your organisation.
 
 ## How to setup a new ODC cluster environment
-Once you have create a terraform backend and updated a configuration parameters, you can perform following steps to setup a new ODC cluster environment -
+Once you have created a terraform backend and updated the configuration parameters, you can perform the following steps to setup a new ODC cluster environment -
 - Change directory to `examples/stage/01_odc_eks/`
 - Run `terraform init` to initialize Terraform state tracking
 - Run `terraform plan` to do a dry run and validate examples and interaction of modules
-- Run `terraform apply` to spin up infrastructure (a new ODC EKS Cluster), can take upto 15-20minutes
+- Run `terraform apply` to spin up infrastructure (a new ODC EKS Cluster), -- can take upto 15-2 0minutes
```

Comment:
little typo here `15-2 0minutes` -> `15-20 minutes`

Reply:
Yes, noticed that for next one, thanks